integration-tests/smoke/auto_ocr_test.go

```diff
@@ -24,25 +24,49 @@ import (
+var defaultOCRRegistryConfig = contracts.KeeperRegistrySettings{
+	PaymentPremiumPPB:    uint32(200000000),
+	FlatFeeMicroLINK:     uint32(0),
+	BlockCountPerTurn:    big.NewInt(10),
+	CheckGasLimit:        uint32(2500000),
+	StalenessSeconds:     big.NewInt(90000),
+	GasCeilingMultiplier: uint16(1),
+	MinUpkeepSpend:       big.NewInt(0),
+	MaxPerformGas:        uint32(5000000),
+	FallbackGasPrice:     big.NewInt(2e11),
+	FallbackLinkPrice:    big.NewInt(2e18),
+	MaxCheckDataSize:     uint32(5000),
+	MaxPerformDataSize:   uint32(5000),
+}
+
+var _ = Describe("Automation OCR Suite @automation", func() {
+	numberOfUpkeeps := 2
 	var (
-		err              error
-		chainClient      blockchain.EVMClient
-		contractDeployer contracts.ContractDeployer
-		registry         contracts.KeeperRegistry
-		registrar        contracts.KeeperRegistrar
-		consumers        []contracts.KeeperConsumer
-		upkeepIDs        []*big.Int
-		linkToken        contracts.LinkToken
-		chainlinkNodes   []*client.Chainlink
-		testEnvironment  *environment.Environment
+		err                  error
+		chainClient          blockchain.EVMClient
+		contractDeployer     contracts.ContractDeployer
+		registry             contracts.KeeperRegistry
+		registrar            contracts.KeeperRegistrar
+		consumers            []contracts.KeeperConsumer
+		consumersPerformance []contracts.KeeperConsumerPerformance
+		performDataChecker   []contracts.KeeperPerformDataChecker
+		upkeepIDs            []*big.Int
+		linkToken            contracts.LinkToken
+		chainlinkNodes       []*client.Chainlink
+		testEnvironment      *environment.Environment
 
 		testScenarios = []TableEntry{
-			Entry("v2.0 Basic smoke test @simulated", ethereum.RegistryVersion_2_0, defaultRegistryConfig, BasicCounter, BasicSmokeTest, big.NewInt(defaultLinkFunds), 2),
+			Entry("v2.0 Basic smoke test @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, BasicSmokeTest, big.NewInt(defaultLinkFunds), numberOfUpkeeps),
+			Entry("v2.0 Add funds to upkeep test @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, AddFundsToUpkeepTest, big.NewInt(1), numberOfUpkeeps),
+			Entry("v2.0 Pause and unpause upkeeps @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, PauseUnpauseUpkeepTest, big.NewInt(defaultLinkFunds), numberOfUpkeeps),
+			Entry("v2.0 Register upkeep test @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, RegisterUpkeepTest, big.NewInt(defaultLinkFunds), numberOfUpkeeps),
+			Entry("v2.0 Pause registry test @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, PauseRegistryTest, big.NewInt(defaultLinkFunds), numberOfUpkeeps),
+			Entry("v2.0 Handle f keeper nodes going down @simulated", ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, BasicCounter, HandleKeeperNodesGoingDown, big.NewInt(defaultLinkFunds), numberOfUpkeeps),
```

Comment:
f=1 so it kind of became a similar test to "Removing one keeper test" so removed that one